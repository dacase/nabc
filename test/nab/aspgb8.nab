// Try some molecular mechanics. check r6 gb

molecule m;
int ier;
float m_xyz[dynamic], f_xyz[dynamic];
float dgrad, fret;

m = getpdb("asp.pdb");
allocate m_xyz[3*m.natoms];
allocate f_xyz[3*m.natoms];

readparm(m, "asp.top");

mm_options("rgbmax=100., cut=100.0, ntpr=100, nsnb=999, gb_debug=1, gb=8, diel=C");
mm_options("gb=8");
mme_init(m, NULL, "::ZZZZ", m_xyz, NULL);
setxyz_from_mol( m, NULL, m_xyz );
mme( m_xyz, f_xyz, -1 );

mm_options("gb_debug=0");

// Do some conjugate gradients.
dgrad = 0.0005;
ier = conjgrad(m_xyz, 3*m.natoms, fret, mme, dgrad, 1.0, 3000);
if(mytaskid==0)
  printf( "conjgrad returns %d\n", ier );

mme_timer();
