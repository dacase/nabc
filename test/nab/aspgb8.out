Reading parm file (asp.top)
title:
ACE 
	mm_options:  rgbmax=100.
	mm_options:  cut=100.0
	mm_options:  ntpr=100
	mm_options:  nsnb=999
	mm_options:  gb_debug=1
	mm_options:  gb=8
	mm_options:  diel=C
	mm_options:  gb=8
Effective Born radii:
17	      1.7000000	      2.3354880
18	      1.5000000	      1.8179126
1	      1.3000000	      1.4820100
9	      1.7000000	      2.3942731
19	      1.5500000	      2.1496444
2	      1.7000000	      2.0391479
10	      1.3000000	      1.7720744
20	      1.3000000	      1.8219150
3	      1.3000000	      1.5120431
11	      1.7000000	      2.3006371
21	      1.7000000	      2.0738430
4	      1.3000000	      1.5185855
12	      1.3000000	      1.7149700
22	      1.3000000	      1.5226333
23	      1.3000000	      1.4965839
24	      1.3000000	      1.5210003
5	      1.7000000	      2.2212187
13	      1.3000000	      1.7082095
6	      1.5000000	      1.8086273
14	      1.7000000	      2.1979939
7	      1.5500000	      2.1705012
15	      1.5000000	      1.7594663
8	      1.3000000	      1.6330212
16	      1.5000000	      1.8217828
     bond:      0.161144365
    angle:      0.420564998
 dihedral:      3.425479213
    enb14:      6.036105088
    eel14:     59.267341807
      enb:     37.461378999
      eel:   -100.091291998
      egb:    -79.420367037
    econs:      0.000000000
    esurf:      0.000000000
    Total:    -72.739644564
     frms:     31.822117358
	mm_options:  gb_debug=0
      iter    Total       bad      vdW     elect   nonpolar   genBorn      frms
ff:     0    -72.74      4.01     43.50    -40.82      0.00    -79.42  3.18e+01
ff:   100   -117.99      5.61      1.99    -44.67      0.00    -80.93  1.03e+00
ff:   200   -118.43      5.73      2.17    -45.36      0.00    -80.97  1.37e-01
ff:   300   -118.66      5.54      1.98    -45.18      0.00    -80.99  3.18e-01
ff:   400   -118.70      5.41      1.86    -44.97      0.00    -81.00  6.07e-02
ff:   500   -118.71      5.45      1.89    -45.08      0.00    -80.97  3.86e-02
ff:   600   -118.71      5.46      1.91    -45.12      0.00    -80.96  1.31e-02
ff:   700   -118.71      5.46      1.91    -45.13      0.00    -80.96  4.99e-03
ff:   800   -118.71      5.46      1.91    -45.13      0.00    -80.96  1.54e-03
conjgrad returns 823

|Timing summary:
|              Initialize       0.000
|             Molec. Dyn.       0.000
|             Normal Mode       0.000
|             Conj. Grad.       0.000
|                  Newton       0.000
|-------------------------
|                   Total       0.000

|1st derivative timing summary:
|             constraints       0.000
|                   bonds       0.002
|                  angles       0.003
|                torsions       0.003
|                pairlist       0.003
|                 nonbond       0.003
|               gen. Born       0.051
|       Poisson Boltzmann       0.000
|                 3D-RISM       0.000
|                   Other       0.003
|-------------------------
|                   Total       0.067

|MPI timing summary:
|             egb reduct.       0.001
|             mme reduct.       0.001
